<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UIAudioBar</class>
 <widget class="QWidget" name="UIAudioBar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>405</width>
    <height>26</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>UIAudioBar</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="styleSheet">
      <string notr="true">border-image: url(:/LoginWindow/images/audioBar.png)</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <property name="leftMargin">
       <number>9</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>9</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label">
        <property name="maximumSize">
         <size>
          <width>75</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>正在录音...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QProgressBar" name="progressBar">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>5</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QProgressBar{
	border-image: url(:/LoginWindow/images/AuxiliaryPanelBack.png);
        border: none;
        border-radius: 2px;}
QProgressBar::chunk{
		border-radius:2px;
        background-color: rgb(7, 157, 210);}</string>
        </property>
        <property name="maximum">
         <number>60</number>
        </property>
        <property name="value">
         <number>0</number>
        </property>
        <property name="textVisible">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="time_label">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="send_pushButton">
        <property name="minimumSize">
         <size>
          <width>45</width>
          <height>22</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>45</width>
          <height>22</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{border-image: url(:/LoginWindow/images/audioSend.png)}
QPushButton:hover{border-image: url(:/LoginWindow/images/audioSend1.png)}</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="cancel_pushButton">
        <property name="minimumSize">
         <size>
          <width>45</width>
          <height>22</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>45</width>
          <height>22</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{border-image: url(:/LoginWindow/images/audioCancel.png)}
QPushButton:hover{border-image: url(:/LoginWindow/images/audioCancel1.png)}</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="loginwindow.qrc"/>
 </resources>
 <connections/>
</ui>
